react/components/EXPERIMENTAL_TableTree/Tree/Node.tsx

```diff
@@ -8,6 +8,17 @@ import CollapseToggle from './CollapseToggle'
 import { Checkboxes } from '../../EXPERIMENTAL_useCheckboxTree/types'
 import { Column } from '../../EXPERIMENTAL_Table'
 import { Density } from '../../EXPERIMENTAL_Table/hooks/useTableMeasures'
+import useTableMotion from '../../EXPERIMENTAL_Table/hooks/useTableMotion'
+
+const TRANSITIONS = [
```

Comment:
Are those the same as `BulkActions/index.tsx`?

Reply:
It was the same as rows, nice catch!